azure_arc_app_services_jumpstart/aks/arm_template/artifacts/AppServicesLogonScript.ps1

```diff
@@ -118,7 +118,7 @@ Do {
 Do {
    Write-Host "Waiting for log-processor to become available. Hold tight, this might take a few minutes..."
    Start-Sleep -Seconds 15
-   $logProcessorStatus = $(if(kubectl describe daemonset ($extensionName + "-k8se-log-processor") -n appservices | Select-String "Pods Status:  4 Running" -Quiet){"Ready!"}Else{"Nope"})
+   $logProcessorStatus = $(if(kubectl describe daemonset ($extensionName + "-k8se-log-processor") -n appservices | Select-String "Pods Status:  3 Running" -Quiet){"Ready!"}Else{"Nope"})
```

Comment:
I find this one weird because i tested it to be working with 4

Reply:
I tried several times and never worked for me with AKS, I had to scale up the cluster to 4 nodes for it to work